package.json

```diff
@@ -26,6 +26,12 @@
     "@nestjs/common": "^8.0.0",
     "@nestjs/core": "^8.0.0",
     "@nestjs/platform-express": "^8.0.0",
+    "@polkadot/api": "^8.8.2",
+    "@polkadot/keyring": "^9.4.1",
+    "@polkadot/types": "^8.8.2",
+    "@polkadot/util": "^9.4.1",
+    "@unique-nft/types": "^0.6.0-4",
```

Comment:
Since yesterday we should replace this library to @unique-nft/unique-mainnet-types": "^0.6.1"

Reply:
Right, done!